Dockerfile.tmpl

```diff
@@ -646,7 +639,11 @@ RUN sed -i '/from tensorflow_hub import uncompressed_module_resolver/a from tens
 #    python -m nb_conda_kernels.install --disable
 
 # Force only one libcusolver
+{{ if eq .Accelerator "gpu" }}
 RUN rm /opt/conda/bin/../lib/libcusolver.so.11 && ln -s /usr/local/cuda/lib64/libcusolver.so.11 /opt/conda/bin/../lib/libcusolver.so.11
+{{ else }}
+RUN ln -s /usr/local/cuda/lib64/libcusolver.so.11 /opt/conda/bin/../lib/libcusolver.so.11
```

Comment:
Same here, I am a bit surprised to see a cuda related command for non-gpu image build... A comment would be helpful.

Reply:
In this one, the build was failing for CPU because the directory it was trying to delete did not exist, but for some reason, that directory did exist on the GPU build, so this was the only way I was able to make it build successfully.

Reply:
Do you remember what process was trying to delete this directory? The underlying issue is likely that we were running a command that shouldn't have been run in the CPU and was missing the `{{ if eq .Accelerator "gpu" }}` guard.

Reply:
Line 643 in the updated file, I didn't change that, I just added the if-else statements around it.